File: DotNetClient/Guts.Client.UI/ApiResult.cs
using System.CodeDom;

namespace Guts.Client.UI
{
    public class ApiResult
    {
        public bool Success { get; set; }
        public string Message { get; set; }

        public static ApiResult CreateSuccess()
        {
            return new ApiResult {Success = true};
        }

        public static ApiResult CreateError(string errorMessage)
        {
            return new ApiResult { Success = false, Message = errorMessage};
        }
    }
}

File: JavaClients/guts-client-core/src/main/java/be/pxl/guts/core/http/ApiResult.java
package be.pxl.guts.core.http;

public class ApiResult {

    private int statusCode;
    private boolean success;
    private String message;

    public ApiResult(int statusCode, boolean success) {
        this(statusCode, success, "");
    }

    public ApiResult(int statusCode, boolean success, String message) {
        this.statusCode = statusCode;
        this.success = success;
        this.message = message;
    }

    public int getStatusCode() {
        return statusCode;
    }

    public boolean isSuccessful() {
        return success;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}
